Studies/python/Optimizer.py

```diff
@@ -31,6 +31,7 @@
 parser.add_argument("-init_points_to_probe", "--init_points_to_probe")
```

Comment:
set `init_points_to_probe` as optional with default value `None` - we don't want to have init points for the second iteration. In the constructor of bo load them only `if init_points_to_probe is not None`.